src/common/comment.rs

```diff
@@ -52,6 +63,13 @@ impl Comment<'_> {
 mod tests {
     use super::*;
 
+    fn as_lintkind_vec(vec: Vec<&str>) -> Vec<LintKind> {
```

Comment:
I don't think that we need a special function in the tests module to convert vectors.
Just use `vec![LintKind::UnorderedKey]`.